docs/examples/cloud_trace_exporter/more_trace.py

```diff
@@ -0,0 +1,43 @@
+from opentelemetry import trace
```

Comment:
Do we want to include this example in the docs? If so, it will need an rst

Reply:
This is from another PR whoops, removing.